vic/vic_run/src/lakes.eb.c

```diff
@@ -26,6 +26,7 @@
 
 #include <vic_def.h>
 #include <vic_run.h>
+#include <vic_driver_shared.h>
```

Comment:
If its required by `vic_run`, its not driver code. Maybe the drivers should handle the creation/initialization of lakes but I think that is a separate issue.